pennylane/__init__.py

```diff
@@ -39,6 +38,7 @@
 from .measure import expval, var, sample, probs
 from .ops import *
 from .optimize import *
+from .templates import *
```

Comment:
How come this got changed from an absolute import?

Reply:
ah it looks like with this change, templates are now top level. The previous `qml.templates.StronglyEntanglingLayers` is now `qml.StronglyEntanglingLayers`. This causes several issues:

1. Do the docs still build/link to the correct places?

2. The tutorials over in the qml repo would need to be updated

3. It causes issues when there are name clashes, i.e., `qml.Interferometer` CV operation and the `qml.Interferometer` subroutine.

I would suggest reverting this back to `import pennylane.templates` to avoid this change in behaviour/UI, as I'm not sure we have enough time to fix it before the upcoming release, and then re-think it before the next release.